code/obj/window.dm

```diff
@@ -1044,6 +1044,28 @@
 
 
 // Flockdrone BS goes here - cirr
+/obj/window/auto/feather
+	default_material = "gnesisglass"
```

Comment:
Was wondering if auto windows somehow inherit from the regular windows, but it seems some stuff is missing like ignoring material appearances?

Reply:
window/auto inherits from /obj/window yes, but it overrides a lot of behaviour